shared_model/interfaces/transaction_responses/tx_response.hpp

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_TX_RESPONSE_HPP
+#define IROHA_TX_RESPONSE_HPP
+
+#include <boost/variant.hpp>
+#include "interfaces/hashable.hpp"
+#include "interfaces/polymorphic_wrapper.hpp"
+#include "interfaces/transaction_responses/committed_tx_response.hpp"
```

Comment:
maybe rename folder to `tx_responses`?

Reply:
Tx not so obvious abbreviation. I guess better use the full name for the folder instead.